apps/company/views.py

```
@@ -27,10 +47,20 @@ class CompanyListView(APIView):
         responses={201: CompanyCreateAndUpdateSerializers()}
     )
     def post(self, request, *args, **kwargs):
-        print('here views')
         serializer = CompanyCreateAndUpdateSerializers(data=request.data, context={'request': request})
         serializer.is_valid(raise_exception=True)
         serializer.save(owner=request.user)
+        update_wildberries_sales.delay()
```

Ruff

`update_wildberries_sales` may be undefined, or defined from star imports

(F405)